### ISSUE TITLE
Batch inference

### ISSUE THREAD

**Issue by elejota1** [psipred/s4pred#5]
I would like to perform batch inference. Can you please point me some resources or provide support for it? Thanks a lot

**Comment by DanBuchan** [comment]
What do you mean by "batch inference" that you want to make s4pred predictions over many individual protein sequences?

**Comment by elejota1** [comment]
Yes! I would like to predict on multiple protein sequences at once as you describe in the "Batched Sequence Prediction" section of the readme.

**Comment by DanBuchan** [comment]
Our colleague is going to implement batch inference shortly. We'll update here once that is done.
